gui/wxpython/gui_core/mapdisp.py

```diff
@@ -160,17 +160,30 @@ def GetToolSwitcher(self):
 
     def SetProperty(self, name, value):
         """Sets property"""
-        if self.HasProperty("projection"):
-            self.statusbarManager.SetProperty(name, value)
-        else:
+        if not self.HasProperty("projection"):
             self.mapWindowProperties.useDefinedProjection = value
+        elif not self.HasProperty("resolution"):
+            self.mapWindowProperties.resolution = value
+        elif not self.HasProperty("alignExtent"):
+            self.mapWindowProperties.alignExtent = value
+        elif not self.HasProperty("region"):
+            self.mapWindowProperties.showRegion = value
+        else:
+            self.statusbarManager.SetProperty(name, value)
 
     def GetProperty(self, name):
         """Returns property"""
-        if self.HasProperty("projection"):
-            return self.statusbarManager.GetProperty(name)
-        else:
+        if not self.HasProperty("projection"):
             return self.mapWindowProperties.useDefinedProjection
+        elif not self.HasProperty("resolution"):
+            return self.mapWindowProperties.resolution
+        elif not self.HasProperty("alignExtent"):
+            return self.mapWindowProperties.alignExtent
+        elif not self.HasProperty("region"):
+            return self.mapWindowProperties.region
+        else:
+            print("bla")
+            return self.statusbarManager.GetProperty(name)
```

Comment:
This doesn't makes sense, I think I missed this during review of #2087. You call e.g. GetProperty("region") and you get useDefinedProjection. See SetProperty above as well.

Reply:
Yes, that's true, thank you @petrasovaa . I was aware of the fact that there is probably a mistake because it crashed but did not have anough time to look at it more precisely.  I will do so now.  You are right that the problem has already started in #2087.